hikari/impl/entity_factory.py

```diff
@@ -3744,3 +3745,81 @@ def deserialize_sku(self, payload: data_binding.JSONObject) -> monetization_mode
             slug=payload["slug"],
             flags=monetization_models.SKUFlags(payload["flags"]),
         )
+
+    ###############
+    # POLL MODELS #
+    ###############
+    def _deserialize_poll_media(self, payload: data_binding.JSONObject) -> poll_models.PollMedia:
+        emoji_payload = payload.get("emoji")
+        return poll_models.PollMedia(
+            text=payload.get("text"), emoji=self.deserialize_emoji(emoji_payload) if emoji_payload else None
+        )
+
+    def _deserialize_poll_answer_count(self, payload: data_binding.JSONObject) -> poll_models.PollAnswerCount:
+        return poll_models.PollAnswerCount(
+            answer_id=payload["answer_id"], count=payload["count"], me_voted=payload["me_voted"]
+        )
```

Comment:
These can be made inline

Reply:
I'm opting to keep the two private `_deserialize`s just in case Discord decides to update them. Just in case though, your opinion might differ.

Reply:
Noted. Resolving.

Reply:
Resolved